yb-voyager/src/dbzm/config.go

```diff
@@ -136,6 +135,12 @@ func (c *Config) String() string {
 	default:
 		panic(fmt.Sprintf("unknown source db type %s", c.SourceDBType))
 	}
+
+	if c.ColumnList != nil {
+		conf += fmt.Sprintf("\ndebezium.source.column.include.list=%s", strings.Join(c.ColumnList, ","))
```

Comment:
Any reason why we didn't go with `column.exclude.list` ?

Reply:
Ora2pg has this conf `MODIFY_STRUCT` which takes the list of table(column_names) where column_names we want to export. So, to have uniform code logic we decided include list here as well.

Reply:
Ah, thought so. Makes sense.